### ISSUE TITLE
does not release original files after repair

### ISSUE THREAD

**Issue by FrisoDB** [anthwlock/untrunc#169]
Using the GUI I have repaired multiple files. Selecting a truncated file and a reference file is easy enough (could be easier with drag and drop instead of scrolling trough big folders). The repair works correctly and a new fixed file is created.

The fixed file can be moved or deleted at will, but the original truncated and reference file cannot. They are locked in place. Closing the whole GUI and restarting it releases the files, but for a workflow it would be way nicer that it just releases the files after the repair.

The analyse tool seems to have the same issue, locking the reference file.

**Comment by Rocketboy235** [comment]
Yep, just experienced this issue. Looks like the program does not properly release the file handle after doing the repair even after you change the file to another video after finishing the first repair.
